src/azure-cli/azure/cli/command_modules/profile/tests/latest/test_auth_e2e.py

```diff
@@ -78,4 +78,5 @@ def test_conditional_access_mfa(self):
         assert decoded['aud'] in scope
         assert decoded['amr'] == ['pwd', 'mfa']
 
+        self.cmd('logout')
```

Comment:
`az logout` already has a mock test:

https://github.com/Azure/azure-cli/blob/b146ace130e07f98a9fcb02ad1036ae67cf806e5/src/azure-cli/azure/cli/command_modules/profile/tests/latest/test_profile_custom.py#L142

Since it is difficult to test it with `ScenarioTest`, we add it to `LiveScenarioTest`.